hwdb.d/60-hid.hwdb

```diff
@@ -0,0 +1,205 @@
+# This file is part of systemd.
+#
+# The lookup keys are composed in:
+#   60-input-id.rules
+#
+# Note: The format of the "hid:" prefix match key is a contract between
```

Comment:
This is a mostly orthogonal to this PR, but I guess we can also discuss this here:
this boilerplate is used in all our files, since forever. But I think it makes no sense. If it were true, users would never be able to make meaningful local hwdb entries. Is there any reason to keep this? I'd prefer not to add it new files as a start.

Reply:
→ https://github.com/systemd/systemd/pull/22990